index.js

```diff
@@ -349,7 +349,7 @@ function fastifyMultipart (fastify, options, done) {
       // don't overwrite prototypes
       if (name in Object.prototype) {
         // ensure that stream is consumed, any error is suppressed
-        sendToWormhole(file)
+        streamToNull(file)
```

Comment:
So this is a Promise (that can reject) and there isn't a `.catch` on it

It can bring everything down

The original package does not reject, or you can add a .catch here

```suggestion
        streamToNull(file).catch(() => {})
```